packages/lit-html/src/directives/choose.ts

```diff
@@ -29,9 +29,9 @@
  * }
  * ```
  */
-export const choose = <T, V>(
+export const choose = <T, V, K extends T = T>(
```

Comment:
Nice! Yeah this fixes the issue. [Playground demo](https://lit.dev/playground/#project=W3sibmFtZSI6InJlcHJvLnRzIiwiY29udGVudCI6ImltcG9ydCB7aHRtbCwgY3NzLCBMaXRFbGVtZW50fSBmcm9tICdsaXQnO1xuaW1wb3J0IHtjdXN0b21FbGVtZW50LCBwcm9wZXJ0eX0gZnJvbSAnbGl0L2RlY29yYXRvcnMuanMnO1xuZGVjbGFyZSBjb25zdCBjaG9vc2U6IDxULCBWLCBLIGV4dGVuZHMgVCA9IFQ-KFxuICB2YWx1ZTogVCxcbiAgY2FzZXM6IEFycmF5PFtLLCAoKSA9PiBWXT4sXG4gIGRlZmF1bHRDYXNlPzogKCkgPT4gVlxuKSA9PiB1bmtub3duO1xuXG50eXBlIENoZWNrb3V0U3RlcCA9ICdyZWdpc3RlcicgfCAnZGVsaXZlcnknIHwgJ3BheW1lbnQnO1xuXG5AY3VzdG9tRWxlbWVudCgneC1yZXBybycpXG5leHBvcnQgY2xhc3MgWFJlcHJvIGV4dGVuZHMgTGl0RWxlbWVudCB7ICBcbiAgcmVuZGVyU3RlcChzdGVwOiBDaGVja291dFN0ZXApIHtcbiAgICByZXR1cm4gY2hvb3NlKHN0ZXAsIFtcbiAgICAgICAgWyd0ZXN0JywgKCkgPT4gMV0sXG4gICAgICAgIFsncmFuZG9tJywgKCkgPT4gMl0sXG4gICAgICAgIFsncmVnaXN0ZXInLCAoKSA9PiAzXSxcbiAgICBdKTtcbiAgfVxuXG4gIHJlbmRlcigpIHtcbiAgICByZXR1cm4gdGhpcy5yZW5kZXJTdGVwKCdyZWdpc3RlcicpO1xuICB9XG59XG4ifSx7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhRE9DVFlQRSBodG1sPlxuPGhlYWQ-XG4gIDxzY3JpcHQgdHlwZT1cIm1vZHVsZVwiIHNyYz1cIi4vcmVwcm8uanNcIj48L3NjcmlwdD5cbjwvaGVhZD5cbjxib2R5PlxuICA8eC1yZXBybz48L3gtcmVwcm8-XG48L2JvZHk-XG4ifSx7Im5hbWUiOiJwYWNrYWdlLmpzb24iLCJjb250ZW50Ijoie1xuICBcImRlcGVuZGVuY2llc1wiOiB7XG4gICAgXCJsaXRcIjogXCJeMi4wLjBcIixcbiAgICBcIkBsaXQvcmVhY3RpdmUtZWxlbWVudFwiOiBcIl4xLjAuMFwiLFxuICAgIFwibGl0LWVsZW1lbnRcIjogXCJeMy4wLjBcIixcbiAgICBcImxpdC1odG1sXCI6IFwiXjIuMC4wXCJcbiAgfVxufSIsImhpZGRlbiI6dHJ1ZX1d)